src/content/guides/getting-started.md

```diff
@@ -258,16 +254,15 @@ bundle.js  544 kB       0  [emitted]  [big]  main
     + 1 hidden module
 ```
 
-T> Custom parameters can be passed to webpack by adding two dashes between the `npm run build` command and your parameters, e.g. `npm run build -- --colors`.
-
+T> We can pass custom parameters to webpack by adding two dashes between the `npm run build` command and your parameters, e.g. `npm run build -- --colors`.
 
 ## Conclusion
 
-Now that you have a basic build together you should move on to the next guide [`Asset Management`](/guides/asset-management) to learn how to manage assets like images and fonts with webpack. At this point, your project should look like this:
+Now that we have a basic build you should move on to the next guide [`Asset Management`](/guides/asset-management) to learn how to manage assets like images and fonts with webpack. At this point, your project should look like this:
```

Comment:
g/you/we
g/your/our

Reply:
I agree with @TheDutchCoder that is better to speak to the reader in _you_ and not as _we_

Reply:
I agree too, but that's what was here before, I was making it consistent across the document. Consistency is the most important, wether it's we/you doesn't matter as much.